File: integration/feature/env-vars/resources/foo/test/src/foo/FooTest.java
package foo;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

public class FooTest {
  @Test
  public void testSimple() {
    String myEnvVar = System.getenv("MY_ENV_VAR");
    System.out.println("test: " + myEnvVar);
    assertEquals(myEnvVar, "abcde");
  }
}

File: example/kotlinlib/simple/5-script/FooTest.kt
//| extends: mill.simple.KotlinModule.Junit5
//| moduleDeps: [Foo.kt]
//| mvnDeps:
//| - "io.kotest:kotest-runner-junit5:5.9.1"
import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.shouldBe

class FooTest :
    FunSpec({
        test("testSimple") {
            generateHtml("hello") shouldBe "<h1>hello</h1>"
        }

        test("testEscaping") {
            generateHtml("<hello>") shouldBe "<h1>&lt;hello&gt;</h1>"
        }
    })
